content/influxdb/cloud/reference/release-notes/cloud-updates.md

```diff
@@ -14,6 +14,31 @@ aliases:
 InfluxDB Cloud updates occur frequently. Find a compilation of recent updates below.
 To find information about the latest Flux updates in InfluxDB Cloud, see [Flux release notes](/influxdb/cloud/reference/release-notes/flux/).
 
+## October 2021
+
+### API invokable scripts
+
+Use [API invokable scripts](/influxdb/cloud/api-guide/api-invocable-scripts/) to assign and execute a script through an API endpoint.
+
+### Clockface v3
+
+Refresh look and feel of InfluxDB Cloud UI. Updated icons, fonts, and layouts to improve visibility, accessibility, and user experience.
+
```

Comment:
do we want to add details here...from the Clockface changelog?

Reply:
> The InfluxDB Cloud UI received a refresh. The updated icons, fonts, and layouts improve visibility, accessibility, and user experience. For those interested in the code, check out the [Clockface UI kit](https://github.com/influxdata/clockface).